### PR TITLE
Alm improvements

### PR DESCRIPTION
**Pull request by nickjameswebb**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scc/delivery-with-flux.hbs.md [hunk 1]
```diff
@@ -1,212 +1,208 @@
...
+  resources: ["ingresses"]
+  verbs: ["get", "list", "create", "update", "delete"]
+```
 
-## Create the PackageInstall
+## Create Carvel PackageInstalls and Secrets
 
-After the `source-to-url-packag` or `basic-image-to-url-package` [supply chains](./ootb-supply-chain-reference.hbs.md) create a Package YAML file in the GitOps repository,
-you must add the packageInstall YAML file in the GitOps repository.
+For each Carvel `Package`, for each Run cluster, you must create a Carvel `PackageInstall` and a `Secret`. The Carvel `PackageInstall` and the `Secret` will be stored in your GitOps repository and deployed to Run clusters by FluxCD.
```
Comment (nader-ziada) [inline scc/delivery-with-flux.hbs.md:42]: Peter asked me in his feedback to specify the supply chain names

File: scc/delivery-with-flux.hbs.md [hunk 2]
```diff
@@ -1,212 +1,208 @@
...
-- To target run clusters from the Build cluster, you must create a secret containing each run cluster Kubeconfig. The FluxCD Kustomization resource uses these secrets.
+- You must create a `Workload` that uses the Carvel Package Supply Chains. See the [documentation](./carvel-package-supply-chain.hbs.md). You must have at least one Carvel `Package` generated by this `Workload` stored in your GitOps repository.
+- You must have at least one Run cluster. Run clusters serve as your deployment environments. They can either be Tanzu Application Platform clusters, or regular Kubernetes clusters, but they must have kapp-controller and Contour installed. See the [Carvel documentation](https://carvel.dev/kapp-controller/) and the [Contour documentation](https://projectcontour.io/).
+- You must create a Build cluster that has network access to your Run clusters. It must have FluxCD Kustomize Controller installed. See the [Flux documentation](https://fluxcd.io/flux/installation/#dev-install).
```
Comment (nader-ziada) [inline scc/delivery-with-flux.hbs.md:12]: In the preview I see this line twice, but here its seems correct, not sure why

Comment (nickjameswebb) [reply scc/delivery-with-flux.hbs.md:12]: Might have something to do with the force push I did

File: scc/delivery-with-flux.hbs.md (hunk 1 above)
Comment (nickjameswebb) [reply scc/delivery-with-flux.hbs.md:42]: I decided to specify the supply chain names up in the prerequisite section, felt a little out of place here.

**Pull request merged**